### PR TITLE
Should support `--config=` and `-c=` styles for configuration file option in Schema Loader

### PR DESCRIPTION
**Pull request by brfrn169**
Please take a look!

### CODE REVIEW

File: schema-loader/src/main/java/com/scalar/db/schemaloader/SchemaLoader.java [hunk 1]
```diff
@@ -67,10 +67,14 @@ static int mainInternal(String... args) {
 
   @Nullable
   private static Object getCommand(String arg) {
+    if (arg.equals("--config")
+        || arg.startsWith("--config=")
+        || arg.equals("-c")
+        || arg.startsWith("-c=")) {
```
Comment (brfrn169) [inline schema-loader/src/main/java/com/scalar/db/schemaloader/SchemaLoader.java:73]: In addition to `--config` and `-c`, We should also allow arguments starting with `--config=` and `-c=`.

**Review by kota2and3kan — APPROVED** [review]
LGTM! Thank you!

I was able to specify the `database.properties` using `--config` with `=` as follows!
* `--config=database.properties`
  ```console
  $ java -jar scalardb-schema-loader-4.0.0-SNAPSHOT.jar --config=database.properties --schema-file=sample-schema.json
  [main] INFO com.scalar.db.schemaloader.command.SchemaLoaderCommand - Config path: database.properties
  [main] INFO com.scalar.db.schemaloader.command.SchemaLoaderCommand - Schema path: sample-schema.json
  [main] INFO com.scalar.db.schemaloader.SchemaOperator - Creating the table t0 in the namespace s0 succeeded.
  ```
* `-c=database.properties`
  ```console
  $ java -jar scalardb-schema-loader-4.0.0-SNAPSHOT.jar -c=database.properties --schema-file=sample-schema.json
  [main] INFO com.scalar.db.schemaloader.command.SchemaLoaderCommand - Config path: database.properties
  [main] INFO com.scalar.db.schemaloader.command.SchemaLoaderCommand - Schema path: sample-schema.json
  [main] INFO com.scalar.db.schemaloader.SchemaOperator - Creating the table t0 in the namespace s0 succeeded.
  ```
* `--config database.properties`
  ```console
  $ java -jar scalardb-schema-loader-4.0.0-SNAPSHOT.jar --config database.properties --schema-file=sample-schema.json
  [main] INFO com.scalar.db.schemaloader.command.SchemaLoaderCommand - Config path: database.properties
  [main] INFO com.scalar.db.schemaloader.command.SchemaLoaderCommand - Schema path: sample-schema.json
  [main] INFO com.scalar.db.schemaloader.SchemaOperator - Creating the table t0 in the namespace s0 succeeded.
  ```
* `-c database.properties`
  ```console
  $ java -jar scalardb-schema-loader-4.0.0-SNAPSHOT.jar -c database.properties --schema-file=sample-schema.json
  [main] INFO com.scalar.db.schemaloader.command.SchemaLoaderCommand - Config path: database.properties
  [main] INFO com.scalar.db.schemaloader.command.SchemaLoaderCommand - Schema path: sample-schema.json
  [main] INFO com.scalar.db.schemaloader.SchemaOperator - Creating the table t0 in the namespace s0 succeeded.
  ```

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**